internal/config/profile.go

```diff
@@ -226,20 +229,65 @@ func (p *profile) SetOrgID(v string) {
 }
 
 // GetConfigDescription returns a map describing the configuration
-func GetConfigDescription(name string) map[string]string {
-	redactedValue := "redacted"
+func GetConfigDescription(name string, redact bool) map[string]string {
+	settings := viper.GetStringMapString(name)
+	newSettings := make(map[string]string)
+
+	fmt.Printf("key %s\n", settings)
+
+	for k, v := range settings {
+		fmt.Printf("key %v val %v\n", k, v)
+		if redact && (k == privateAPIKey || k == publicAPIKey) {
+			newSettings[k] = "redacted"
+		} else {
+			newSettings[k] = v
+		}
+	}
+
+	return newSettings
+}
+
+// DeleteConfig deletes an existing configuration
+func DeleteConfig(name string) error {
```

Comment:
this approach works for the delete command, I'd even suggest let's do that first since it feels simpler than rename right now, but writing and reloading from disk is a bit slow, specially doing it multiple times for a single comamnd, I'd like rename to do all in memory in one go, find, delete, replace

Reply:
I think that might be the easier solution -- I'll make a delete command now based on this. Rename might be easier in a similar way, just editing the toml directly